src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,11 @@
+function ArticleCard({ article }) {
+  return (
+    <section className = "article-card">
+      <img src={article.article_img_url} alt="article image" />
+      <h2>{article.title}</h2>
+      <h3>{article.author}</h3>
+    </section>
```

Comment:
A bit picky, but it would be nice to see labels for your users. Like Author: jessjelly. There is also more info on the article object that you can use, like created_at and votes. Both of which could come in handy to see the results of your work on a future sorting articles ticket 😄